### DISCUSSION TITLE
Offline next.js documentation

### DISCUSSION THREAD

**Discussion by maayanbar13** [vercel/next.js#26960] [Help]
Is there any way to build your own next.js documentation site?   
There is no clear documentation how to turn the /docs folder into the next.js documentation site.   
Is it proprietary?

**Comment by naveennamani** [comment]
I'm having the same issue, it would be great to have the documentation working offline, so that it can be referred multiple times during development. Any solution for this @maayanbar13 ?

**Comment by eigenpaul** [comment]
The /docs folder only holds the markdown files, from which the documentation site then gets build. The source code for their docs is not public and I don't think they are interested in publishing it anyways. So in a sense, yes it's proprietary.
